### ISSUE TITLE
Not working for VSCODE this is essential

### ISSUE THREAD

**Issue by fordneild** [rxhanson/Rectangle#182]
macOS version: 10.15.5
Rectangle version:
logs (In Rectangle menu, hold option, "View Logging..."): 
2020-06-25T18:20:18-04:00: Unable to obtain accessibility element
2020-06-25T18:20:18-04:00: Unable to obtain accessibility element
2020-06-25T18:20:19-04:00: Unable to obtain accessibility element
2020-06-25T18:20:19-04:00: Unable to obtain accessibility element
2020-06-25T18:20:19-04:00: Unable to obtain accessibility element

**Comment by rxhanson** [comment]
Did you try restarting VSCode as noted in https://github.com/rxhanson/Rectangle/issues/167 and https://github.com/rxhanson/Rectangle/issues/43?

**Comment by rxhanson** [comment]
Closing this one out, as there's nothing that can be done from Rectangle's perspective as far as I know. Let me know if a VSCode restart doesn't affect things for you, though.
